src/Form/Control/DropdownCascade.php

```diff
@@ -65,7 +65,7 @@ public function getNewValues(string $id): array
         $model = $this->cascadeControl->model->load($id)->ref($this->reference);
         $values = [];
         foreach ($model as $k => $row) {
-            if ($this->renderRowFunction && is_callable($this->renderRowFunction)) {
+            if ($this->renderRowFunction) {
                 $res = call_user_func($this->renderRowFunction, $row, $k);
```

Comment:
`$res = $this->renderRowFunction($row, $k);`?

Reply:
https://github.com/atk4/ui/pull/1384#discussion_r462926547